go/mysql/auth_server.go

```diff
@@ -277,10 +277,10 @@ func ScrambleMysqlNativePassword(salt, password []byte) []byte {
 	return scramble
 }
 
-// DecodeMysqlNativePasswordHex decodes the standard format used by MySQL
+// DecodePasswordHex decodes the standard format used by MySQL
 // for 4.1 style password hashes. It drops the optionally leading * before
 // decoding the rest as a hex encoded string.
-func DecodeMysqlNativePasswordHex(hexEncodedPassword string) ([]byte, error) {
+func DecodePasswordHex(hexEncodedPassword string) ([]byte, error) {
```

Comment:
Renamed this since it's really a hex decoding helper, not specifically related to `mysql_native_password`.